### PR TITLE
Make the Patient Search Remember the Previous Search

### PR DESCRIPTION
**Pull request by enyachoke**

### CODE REVIEW

**Comment by Werick** [comment]
What happened to the index page? I thought you are adding only the new service.

File: app/scripts/patient-search/services/patient-search.service.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*jshint -W003, -W117, -W098, -W026 */
+(function() {
```
Comment (Werick) [inline app/scripts/patient-search/services/patient-search.service.js]: Look at some our services and follow the same structure. We can discuss this

**Comment by enyachoke** [comment]
@Werick  I think my text editor reformats code when I save a file. Has it caused any issues

File: app/scripts/patient-search/services/patient-search.service.js (hunk 1 above)
Comment (enyachoke) [inline app/scripts/patient-search/services/patient-search.service.js]: Should be okay now

File: app/scripts/clinic-dashboard/controllers/clinic-dashboard.service.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/*jshint -W003, -W117, -W098, -W026 */
```
Comment (Werick) [inline app/scripts/clinic-dashboard/controllers/clinic-dashboard.service.js]: This Service is tied to which ticket?

Comment (enyachoke) [inline app/scripts/clinic-dashboard/controllers/clinic-dashboard.service.js]: It holds data necessary for maintaining state and will be used to achieve AA-253 which I have done and AA-251 which am currently doing

Comment (Werick) [inline app/scripts/clinic-dashboard/controllers/clinic-dashboard.service.js]: Since it is a service  kindly follow the same structure for other services

**Pull request closed without merging**